plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -202,6 +202,7 @@ def get_custom(self, settings: SerializationSettings) -> Optional[Dict[str, Any]
                 max_restarts=self.task_config.max_restarts,
             )
             job = DistributedPyTorchTrainingTask(
+                workers=self.task_config.nnodes,
```

Comment:
Notice how in [this](https://github.com/kubeflow/training-operator/blob/master/examples/pytorch/elastic/echo/echo.yaml) example the number of replicas is set independently of the elastic config.